src/hooks/useTaskProgress.tsx

```diff
@@ -0,0 +1,56 @@
+import { useMemo, useCallback } from 'react';
+import { Newbie } from 'buddy-app-schema';
+import { isCompleted } from 'utils';
+
+interface TaskProgress {
+  getBuddyProgress: (newbie: Newbie) => number;
+  buddyProgress: number;
+  newbieProgress: number;
+}
+
+const transformNewbieProgress = (newbie?: Newbie): number => {
+  if (newbie) {
+    const tasks = newbie.newbieTasks;
+    const tasksAmount = tasks.length;
+    const tasksCompleted = tasks.filter(task => task && isCompleted(task.status))
```

Comment:
It's minor, but I assume that because `task` could be `null` you had to add `task && isCompleted(task.status)`, right?

This led me to following improvement:
1. Change slightly implementation of `isCompleted` function:
https://github.com/BuildIt-Poland/buddy-app/blob/a9bed59e436f3330ddf7d98e890f04645711bf20/src/utils/helpers.ts#L6

to 
```
export const isCompleted = (task: Maybe<NewbieTask>) => task && task.status === TaskStatus.Completed;
```

2. Now you can simplify `task.filter()` occurrences
https://github.com/BuildIt-Poland/buddy-app/blob/a9bed59e436f3330ddf7d98e890f04645711bf20/src/hooks/useTaskProgress.tsx#L15
https://github.com/BuildIt-Poland/buddy-app/blob/a9bed59e436f3330ddf7d98e890f04645711bf20/src/hooks/useTaskProgress.tsx#L29
https://github.com/BuildIt-Poland/buddy-app/blob/a9bed59e436f3330ddf7d98e890f04645711bf20/src/hooks/useTaskProgress.tsx#L32
to
```
    const newbieTasksCompleted = tasksNewbie.filter(isCompleted).length;
    const buddyTasksCompleted = tasksBuddy.filter(isCompleted).length;
```

Reply:
Fixed (here)[https://github.com/BuildIt-Poland/buddy-app/pull/61/commits/3e02a6abae9f8d5047bbbd438ed50029014a4ad9]

Reply:
Hmmm, I see that `isCompleted` is still inside `utils.ts` - do you use it anywhere else or plan to use it? I wonder why it's splitted?   Isn't it a little bit redundant to have both functions `isCompleted` and `isCompletedTask`? I mean, it could be 
simplified to only use `isCompletedTask`. But maybe I'm not seeing something?

```
export const isCompleted = (task: Maybe<NewbieTask>) => task && task.status === TaskStatus.Completed;
```